src/rust/src/x509.rs

```diff
@@ -945,9 +1441,6 @@ fn parse_x509_extension(py: pyo3::Python<'_>, der_oid: &[u8], ext_data: &[u8]) -
             .getattr("PolicyConstraints")?
             .call1((pc.require_explicit_policy, pc.inhibit_policy_mapping))?
             .to_object(py))
-    } else if oid == *PRECERT_POISON_OID {
```

Comment:
Should this function be renamed `parse_csr_extension` given that that's not what it's for?

Reply:
We can rename it, sure. As we convert CSR we'll rip this out entirely but can't do that yet so might as well name it accurately.